src/Commands/Backup/Automatic/EnableCommand.php

```diff
@@ -33,6 +33,12 @@ class EnableCommand extends TerminusCommand implements SiteAwareInterface
     public function enableSchedule($site_env, $options = ['day' => null, 'keep-for' => null,])
     {
         list(, $env) = $this->getSiteEnv($site_env);
+
+        // Schedule Backup workflow expects 'weekly-ttl' as input option.
+        if(isset($options['keep-for']) && !is_null($options['keep-for'])) {
+            $options['weekly-ttl'] = $options['keep-for'];
```

Comment:
Is our minimum php version high enough that we could simply use `$options['weekly-ttl'] ?= $options['keep-for'];`?

Will probably want to keep this version when backporting to Terminus 2.

Reply:
Ah, I saw the PR was against master, which I presumed was 3.x-dev. Usually, the process is to do all development on the latest branch, and then backport to the legacy branch once it's working on latest. Maybe now would be a good time to rename `master` to `2.x`, and only use `2.x` and `3.x` as the dev branches moving forward.

Reply:
As discussed in slack, `??=` does not really help here, as PHP still emits a warning if the RHS array key element does not exist. So, AFAIK, the code as written here is probably the best we can do for PHP 7.4+ on the 3.x branch too.

Reply:
@namespacebrian `!is_null` in the `if` statement is redundant - `if(isset($options['keep-for']))` would be enough.

Reply:
Yeah I discovered when testing yesterday (after creating the PR), that
`isset($array['some_key'])` will evaluate false if that key is defined but
the value is null.
I'd basically copied that code from here:
https://github.com/pantheon-systems/terminus/blob/43d6065e574172fb1fea4214eea432b5bb089ce8/src/Collections/Backups.php#L231
`isset()` seems to be behaving differently from how I thought I
remembered.. I also thought that `isset($array['some_key'])` would cause a
notice if "some_key" didn't exist and had been checking
`array_key_exists()` to avoid a notice, I thought...

Reply:
Yeah, `isset` will be quiet if the key does not exist. Simplifying the if is less important than removing `if` statements where possible, but since that's not possible, go ahead and simplify the clause if you want. Optional.